pm/sender.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/pkg/errors"
 )
 
+// ErrSenderValidation is emitted when sender.validateSender() encounters an error
```

Comment:
In case `ErrSenderValidation` is ever returned from a function that is not called `validateSender()` (i.e. if there was a refactor), we could drop `sender.validateSender()` from this comment and just say something like:

```
ErrSenderValidation is returned when the sender cannot send tickets
```